sunbeam-python/sunbeam/commands/openstack.py

```diff
@@ -334,6 +334,18 @@ def run(self, status: Optional[Status] = None) -> Result:
                 ),
             }
         )
+
+        # NOTE(jamespage)
+        # When dedicated control nodes are used, ceph is not enabled during
+        # bootstrap - however storage nodes may be added later so re-assess
+        # as to whether ceph integrations can be enabled during resize.
+        storage_nodes = client.cluster.list_nodes_by_role("storage")
```

Comment:
storage_nodes is already initialised @ https://github.com/canonical/snap-openstack/blob/main/sunbeam-python/sunbeam/commands/openstack.py#L326